dotnet/src/SemanticKernel.Abstractions/SemanticKernel.Abstractions.csproj

```diff
@@ -28,5 +28,6 @@
     <InternalsVisibleTo Include="SemanticKernel.Extensions.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.SemanticKernel.Planners.Core.UnitTests" />
     <InternalsVisibleTo Include="DynamicProxyGenAssembly2" /> <!-- Moq -->
+    <InternalsVisibleTo Include="Microsoft.SemanticKernel.Experimental.Assistants" />
```

Comment:
Nothing to change now but we should revisit adding this to the abstractions and core projects. What is it you need that is internal?

Reply:
I think it was testing related...agree to revisist